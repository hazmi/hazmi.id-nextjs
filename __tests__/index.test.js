import React from 'react'
import renderer from 'react-test-renderer'
import Index from '../pages/index'

describe('With Snapshot Testing', () => {
  it('App shows "Hello"', () => {
    const component = renderer.create(<Index />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})