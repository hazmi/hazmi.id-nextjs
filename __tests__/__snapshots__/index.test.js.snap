// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing App shows "Hello" 1`] = `
<section
  className="intro"
>
  <p>
    Hello!
  </p>
  <p>
    My name is 
    <strong>
      Hazmi Abdun Nazir
    </strong>
    . I create mobile apps and websites, as a hobby and for a living.
  </p>
  <section>
    <h3>
      Contact
    </h3>
    <p>
      If you need anything, I'm on 
      <a
        aria-label="Go to Hazmi's LinkedIn"
        href="https://www.linkedin.com/in/hazmi-abdun-nazir/"
      >
        LinkedIn
      </a>
      , 
      <a
        aria-label="Go to Hazmi's Github"
        href="https://github.com/hazmi"
      >
        GitHub
      </a>
       or 
      <a
        aria-label="Go to Hazmi's Twitter"
        href="https://twitter.com/hazmi"
      >
        Twitter
      </a>
      .
    </p>
  </section>
  <section>
    <h3>
      About this site
    </h3>
    <p>
      This site is built using 
      <a
        aria-label="Go to Next.js website"
        href="https://nextjs.org/"
      >
        Next.js
      </a>
      , a 
      <a
        aria-label="Go to React Website"
        href="https://reactjs.org/"
      >
        React
      </a>
       Framework. This site is still in progress, currently optimized for mobile.
    </p>
    <p>
      <em>
        [will write more about the site]
      </em>
    </p>
  </section>
</section>
`;
